src/common/utility.ts

```diff
@@ -403,7 +403,8 @@ export const updateURLAsPerUserLanguage = () => {
     const paths = current_path.split('/')
     const first_path = paths[1]
     const has_language_in_url = first_path in language_config
-    has_language_in_url && Cookies.set('user_language', first_path)
+    document.cookie =
+        has_language_in_url && 'user_language=' + first_path + ';domain=.' + getDomain() + ';path=/'
```

Comment:
Is there a library that can handle this or can we find any existing helpers that can do thesame thing? Otherwise let's create our own in the `helpers` directory to keep this clean.